"typings" field in package.json is broken

the `typings` field was added by
https://github.com/mozilla/source-map/commit/6f5d5946d022312f41433c8f847635427ed2e854
which also added `source-map.d.ts`, however that file wasn't included in the published package:
https://unpkg.com/source-map@0.5.7/

```
foo.ts(1,21): error TS7016: Could not find a declaration file for module 'source-map'. '/usr/local/google/home/alexeagle/Projects/repro_source-map/node_modules/source-map/source-map.js' implicitly has an 'any' type.
  Try `npm install @types/source-map` if it exists or add a new declaration (.d.ts) file containing `declare module 'source-map';`
```

/cc @tbosch

--- a/package.json
+++ b/package.json
@@ -49,6 +49,7 @@
   "main": "./source-map.js",
   "files": [
     "source-map.js",
+    "source-map.d.ts",
     "lib/",
     "dist/source-map.debug.js",
     "dist/source-map.js",

